pylab/website/static/css/main.scss

```diff
@@ -38,3 +38,8 @@
         margin-top: 20px;
     }
 }
+.logo {
+	margin-right: 5px;
+	margin-left: 5px;
+	float: left;
+}
```

Comment:
Instead of adding your own styling please use bootstrap styles if possible.

In this particular case, you should use http://getbootstrap.com/components/#navbar-brand-image